src/library_exceptions.js

```diff
@@ -365,10 +366,10 @@ var LibraryExceptions = {
 #if EXCEPTION_DEBUG
         err("  __cxa_find_matching_catch found " + [ptrToString(info.get_adjusted_ptr()), caughtType]);
 #endif
-        {{{ makeStructuralReturn(['thrown', 'caughtType']) }}};
+        return caughtType;
       }
     }
-    {{{ makeStructuralReturn(['thrown', 'thrownType']) }}};
```

Comment:
It looks like this flips the arguments. The `return` should be of the first item, `thrown`, and `setTempRet0(thrownType)`.

Reply:
Ooops!  Fixed